dbms/src/Interpreters/ClusterProxy/TruncateStreamFactory.cpp

```diff
@@ -0,0 +1,68 @@
+#include <Interpreters/ClusterProxy/TruncateStreamFactory.h>
```

Comment:
We shouldn't proxy `TRUNCATE` query for Distributed table, because query `ON CLUSTER` already do it.